### ISSUE TITLE
Single image not working

### ISSUE THREAD

**Issue by huubl** [wpmetabox/mb-elementor-integrator#16]
Hi,

I have an image advanced field limited to  one image. But when selecting the image field in Elementor it's not showing up.

Settings:
```
array(
    'id' => 'header-image',
    'type' => 'image_advanced',
    'name' => 'image',
    'max_file_uploads' => '1',
    'class' => 'image-header-upload',
),
```

var_dump output of [$images = $this->handle_get_value();](https://github.com/wpmetabox/mb-elementor-integrator/blob/dd828817f25445a34798bc5035e78097454ea7f6/src/Traits/Fields/Image.php#L15) 
```
Array (
    [20161] => Array (
            [width] => 150
            [height] => 150
            [file] => 2022/07/banner.jpg
            ....
            [ID] => 20161
            [name] => banner-scaled-e1650976727162.jpg
            ....
        )
)
```
According to the output above $images['ID'] won't work:

https://github.com/wpmetabox/mb-elementor-integrator/blob/dd828817f25445a34798bc5035e78097454ea7f6/src/Traits/Fields/Image.php#L21-L27

It works when replaced with:

```
// Single image.
if ( count( $images) === 1 ) {
	$images = $images[array_key_first($images)];

	return [
		'id'  => $images['ID'],
		'url' => $images['full_url'],
	];
}
```

**Comment by huubl** [comment]
Pull request: Fix single image #22

**Comment by rilwis** [comment]
Thanks for your help and support!
